src/Service/ServiceListenerFactory.php

```diff
@@ -1,16 +1,56 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Laminas\Mvc\Service;
 
-use Laminas\Mvc\View\Http\DefaultRenderingStrategy;
-use Interop\Container\ContainerInterface;
+use interop\container\containerinterface;
 use Laminas\ModuleManager\Listener\ServiceListener;
 use Laminas\ModuleManager\Listener\ServiceListenerInterface;
 use Laminas\Mvc\Controller\ControllerManager;
-use Laminas\Mvc\View;
+use Laminas\Mvc\Controller\PluginManager;
```

Comment:
Sorry to be a pain in the ass here, but you you instead `use Laminas` and just remove the leading `\` where used - my reasoning is that the import list gets massive in a file like this and when scanning files like this, it's more helpful to see the namespace inline

Reply:
sorry but this request change is verrryyy annoying :s, could you do it by yourself once it's merged ?

Reply:
Nope. You started this 😜

Reply:
@fezfez 
I agree with @gsteel here. It would be great if you could still make this change.
Thanks in advance! 👍🏻

Reply:
@gsteel @froschdesign i've done something in between, let me know if it's ok for you :)